core/utils.py

```diff
@@ -49,3 +51,12 @@ def get_approximate_date(date_str):
         except ValueError:
             return None
     return None
+
+def next_deadline():
+    """Return the next deadline for GitHub sponsoring so the organizers know when we will send their request."""
+    now = datetime.now()
```

Comment:
Because we're only working with dates (not datetimes), you can use `today = date.now()` (`from datetime import date`).